Added possibility of passing the parameters with a file (#96)
vmware-labs/vmware-image-builder-action: README.md

The above line is using the GitHub Action default input parameters. You can customize those parameters if you need to, and in fact, this will be pretty common when you have multiple pipelines that need to be sent to VIB:

> | Attribute               | Description                                                                                                                                         | Default value                |

Reviewer: I think you are reordering the attributes and they were alphabetically ordered. Is it intentional?

Reviewer: Yes. I think it's not in alphabetical order with your change (note the runtime-parameters-file attribute)

| ------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
| config                  | This is the default folder where the action can find the configuration files for the different tasks that will be executed as part of the pipeline. | .vib                         |
| backoff-intervals       | This is the default backoff time (in milliseconds) used between each retry in case of failure reaching out to VIB.                                  | [5000, 10000, 15000]         |

> | runtime-parameters-file | Ths parameter set the location of the file with the rutine parameters.                                                                              | runtime-parameters-file.yaml |

Reviewer 2: ```suggestion
| runtime-parameters-file | This parameter specifies the location of the file with the runtime parameters in plain text.                                                                              | runtime-parameters-file.yaml |
```
